src/main/java/se/sundsvall/messaging/api/StatusAndHistoryResource.java

```diff
@@ -62,59 +76,43 @@ class StatusAndHistoryResource {
-		@ApiResponse(responseCode = "400", description = "Bad Request", content = @Content(schema = @Schema(oneOf = {Problem.class, ConstraintViolationProblem.class}))),
-		@ApiResponse(responseCode = "500", description = "Internal Server Error", content = @Content(schema = @Schema(implementation = Problem.class)))
-	})
+	@Operation(summary = "Get the entire conversation history for a given party")
 	@GetMapping(value = CONVERSATION_HISTORY_PATH, produces = {APPLICATION_JSON_VALUE, APPLICATION_PROBLEM_JSON_VALUE})
 	ResponseEntity<List<HistoryResponse>> getConversationHistory(
 		@Parameter(name = "municipalityId", description = "Municipality id", example = "2281") @ValidMunicipalityId @PathVariable final String municipalityId,
 		@Parameter(schema = @Schema(format = "uuid")) @PathVariable @ValidUuid final String partyId,
 		@RequestParam(name = "from", required = false) @DateTimeFormat(iso = DateTimeFormat.ISO.DATE) @Parameter(description = "From-date (inclusive). Format: yyyy-MM-dd (ISO8601)") final LocalDate from,
 		@RequestParam(name = "to", required = false) @DateTimeFormat(iso = DateTimeFormat.ISO.DATE) @Parameter(description = "To-date (inclusive). Format: yyyy-MM-dd (ISO8601)") final LocalDate to) {
 
-		return ResponseEntity.ok(historyService.getConversationHistory(municipalityId, partyId, from, to).stream()
+		return ok(historyService.getConversationHistory(municipalityId, partyId, from, to).stream()
 			.map(ApiMapper::toHistoryResponse)
 			.toList());
 	}
 
-	@Operation(summary = "Get the status for a message batch, its messages and their deliveries", responses = {
-		@ApiResponse(responseCode = "200", description = "Successful Operation", useReturnTypeSchema = true),
-		@ApiResponse(responseCode = "400", description = "Bad Request", content = @Content(schema = @Schema(oneOf = {Problem.class, ConstraintViolationProblem.class}))),
-		@ApiResponse(responseCode = "404", description = "Not Found", content = @Content(schema = @Schema(implementation = Problem.class))),
-		@ApiResponse(responseCode = "500", description = "Internal Server Error", content = @Content(schema = @Schema(implementation = Problem.class)))
-	})
+	@Operation(summary = "Get the status for a message batch, its messages and their deliveries")
+
 	@GetMapping(value = BATCH_STATUS_PATH, produces = {APPLICATION_JSON_VALUE, APPLICATION_PROBLEM_JSON_VALUE})
 	ResponseEntity<MessageBatchResult> getBatchStatus(
 		@Parameter(name = "municipalityId", description = "Municipality id", example = "2281") @ValidMunicipalityId @PathVariable final String municipalityId,
 		@Parameter(schema = @Schema(format = "uuid")) @PathVariable @ValidUuid final String batchId) {
 
 		final var history = historyService.getHistoryByMunicipalityIdAndBatchId(municipalityId, batchId);
-		return history.isEmpty() ? ResponseEntity.notFound().build() : ResponseEntity.ok(toMessageBatchResult(history));
+		return history.isEmpty() ? ResponseEntity.notFound().build() : ok(toMessageBatchResult(history));
```

Comment:
Import notFound static (same as you have done with ok)? Comment goes for all places where ResponseEntity still is used as prefix.